server/channels/app/webhook.go

```diff
@@ -95,21 +95,38 @@ func (a *App) handleWebhookEvents(c request.CTX, post *model.Post, team *model.T
 }
 
 func (a *App) TriggerWebhook(c request.CTX, payload *model.OutgoingWebhookPayload, hook *model.OutgoingWebhook, post *model.Post, channel *model.Channel) {
-	var body io.Reader
-	var contentType string
+
+	var jsonBytes []byte
+	var err error
+
+	contentType := "application/x-www-form-urlencoded"
 	if hook.ContentType == "application/json" {
-		js, err := json.Marshal(payload)
+		contentType = "application/json"
+		jsonBytes, err = json.Marshal(payload)
 		if err != nil {
 			c.Logger().Warn("Failed to encode to JSON", mlog.Err(err))
+			return
 		}
-		body = bytes.NewReader(js)
-		contentType = "application/json"
-	} else {
-		body = strings.NewReader(payload.ToFormValues())
-		contentType = "application/x-www-form-urlencoded"
 	}
 
 	for i := range hook.CallbackURLs {
+		var body io.Reader
+		if hook.ContentType == "application/json" {
+			body = bytes.NewReader(jsonBytes)
+		} else {
+			body = strings.NewReader(payload.ToFormValues())
+		}
+			js, err := json.Marshal(payload)
+			if err != nil {
+				c.Logger().Warn("Failed to encode to JSON", mlog.Err(err))
+			}
+			body = bytes.NewReader(js)
+			contentType = "application/json"
+		} else {
+			body = strings.NewReader(payload.ToFormValues())
+			contentType = "application/x-www-form-urlencoded"
+		}
```

Comment:
@daran9 looks like there's a bad merge here. Can you please check and correct this. 

```suggestion
		js, err := json.Marshal(payload)
		if err != nil {
			c.Logger().Warn("Failed to encode to JSON", mlog.Err(err))
		}
		body = bytes.NewReader(js)
```

Reply:
@srkgupta, it looks good to me. Can you try viewing it without whitespace changes?

![CleanShot 2023-04-17 at 10 17 38](https://user-images.githubusercontent.com/6913320/232513423-b272aeee-6385-4f8f-9893-1c9cb37e1f3f.png)

Reply:
@mickmister I am still seeing the error. I even confirmed the error by checking out the branch and viewing the code in VSCode. If you notice that the `else` block in the screenshot shown below has error:

<img width="771" alt="Screenshot 2023-04-18 at 11 09 07 AM" src="https://user-images.githubusercontent.com/1429138/232681654-f15c1ff2-0e66-4d3a-9f20-1d89815020c5.png">

https://github.com/mattermost/mattermost-server/pull/22394/files?diff=unified&w=1#diff-fd166b8fa6e5523c4acf7c56ccc49b2b0d94c861e1b0d31804b9cb463be1ee18R125-R128

Reply:
ah yes! I believe it's fixed now. Thanks @srkgupta